docs/cli/introduction.md

```diff
@@ -1,42 +1,42 @@
 ---
 title: An introduction to Terramate CLI
 description: Terramate CLI is an open-source Infrastructure as Code (IaC) orchestration tool for Terraform, OpenTofu, Terragrunt, Kubernetes, Pulumi, AWS Cloud Formation, AWS Cloud Development Kit (CDK), Azure Resource Manager (ARM), Biceps, and others.
-
-next:
-  text: 'Installation'
-  link: '/cli/installation'
 ---
 
 # Introduction
 
 This page provides a high-level overview of what Terramate CLI is and how it works.
 
-If you want to get started with a *practical introduction* and learn about the Terramate CLI API, head over to the
+If you want to get started with a practical introduction and learn about the Terramate CLI API, head over to the
 [Getting Started](./getting-started/index.md) guide.
```

Comment:
On the nav it is called the "Quickstart". We should rename that, or or this. One or the other. It's a small detail but helps with continuity.

Reply:
quickstart will be handled in a follow up PR. thanks for the catch!